MANGA.py

```diff
@@ -52,6 +51,13 @@ def main(argv):
             sys.exit()
         elif opt in ("-i", "--project_file"):
             project_file = str(arg)
+        elif opt in ("-l", "--project_file"):
```

Comment:
I do not understand this. What happens, if MANGA is executed using
`python MANGA.py --project_file blabliblubb`

Shouldn't this line be 
```suggestion
        elif opt in ("-l", "--logging"):
```

Reply:
my_project_file = dummy/for/path/to/my/project/file

Reply:
solved